manifests/s/Silversoft/speed/8.60/Silversoft.speed.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Silversoft.speed
+PackageVersion: "8.60"
+InstallerType: portable

```

Comment:
This is not a portable application. The exe is a setup file which needs to be run to install the application